src/services/course-cooperation-service.ts

```diff
@@ -1,14 +1,20 @@
-import { AxiosResponse } from 'axios'
-import { URLs } from '~/constants/request'
-import { axiosClient } from '~/plugins/axiosClient'
 import { CourseCooperationResponse } from '~/types'
-import { createUrlPath } from '~/utils/helper-functions'
+import { URLs } from '~/constants/request'
+import { baseService } from '~/services/base-service'
+import { getFullUrl } from '~/utils/get-full-url'
 
 export const CoursesAndCooperationsService = {
   getByResourceId: async (
     resourceId: string
-  ): Promise<AxiosResponse<CourseCooperationResponse>> =>
-    await axiosClient.get(
-      createUrlPath(URLs.coursesAndCooperations.getByResourceId, resourceId)
-    )
+  ): Promise<CourseCooperationResponse> => {
```

Comment:
You can remove type from here, it's already typed correctly by `baseService`
```suggestion
  ) => {
```

Reply:
Done.